plugins/woocommerce/tests/e2e-pw/tests/merchant/lost-password.spec.js

```diff
@@ -0,0 +1,38 @@
+const { test, expect } = require( '@playwright/test' );
+const { admin } = require( '../../test-data/data' );
+
+test.describe( 'Can go to lost password page and submit the form', () => {
+	test( 'can visit the lost password page from the login page', async ( {
+		page,
+	} ) => {
+		await page.goto( '/wp-login.php' );
+		await page.getByRole( 'link', { name: 'Lost your password?' } ).click();
+		expect( page.url() ).toContain( '/wp-login.php?action=lostpassword' );
+		await expect( page.getByText( 'Get New Password' ) ).toBeVisible();
+	} );
+
+	test( 'can submit the lost password form', async ( { page } ) => {
+		await page.goto( '/wp-login.php?action=lostpassword' );
+		await page.getByLabel( 'Username or Email Address' ).click();
+		await page
+			.getByLabel( 'Username or Email Address' )
+			.fill( admin.username );
+		await page.getByRole( 'button', { name: 'Get New Password' } ).click();
+
+		try {
+			await page.waitForURL( '**/wp-login.php?checkemail=confirm' );
+			await expect(
+				page.getByText( /Check your email for the confirmation link/i )
+			).toBeVisible();
```

Comment:
@chihsuan  Sorry for another comment post-merge 😅 

I think since it's more common to have the email sending fails, we should swap the logic since otherwise this causes unnecessary test delays to wait for timeout in CI

Reply:
No worries! I agree with you. I'll make the change

Reply:
Created PR here https://github.com/woocommerce/woocommerce/pull/50666